src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
remove empty line

Reply:
if I remove empty line, I have error "'CLASS_DEF' should be separated from previous statement. [EmptyLineSeparator]"